src/config_cache.cpp

```diff
@@ -409,7 +409,7 @@ bool config_cache::delete_cache_files(const std::vector<std::string>& paths,
 }
 
 config_cache_transaction::state config_cache_transaction::state_ = FREE;
-config_cache_transaction* config_cache_transaction::active_ = 0;
+config_cache_transaction* config_cache_transaction::active_ = nullptr;
```

Comment:
What on earth is `= FREE`…?

Reply:
Very good question. I have no idea, I didn't write that line.

Reply:
I assume it's defined as a macro or a constant somewhere in the source…

Reply:
It's an enum. I think it means that by default, there is no active cache transaction (i.e. nullptr) and the enum means it's free to take another transaction.

Reply:
Ah, that makes sense.